lib/instadeploy.py

```diff
@@ -169,3 +187,13 @@ def build():
         ))
 
     return response.json()
+
+
+def extract_mxbuild_response(mxbuild_response):
+    r = {}
+    if mxbuild_response and 'problems' in mxbuild_response:
+        r['buildstatus'] = json.dumps(mxbuild_response['problems'])
+        if 'message' in mxbuild_response:
```

Comment:
I think this should have the same check and at the same level as `problems` and not nested inside:

```python
if mxbuild_response and 'message' in mxbuild_response:
```